src/Calendar/Calendar.js

```diff
@@ -659,12 +714,14 @@ Calendar.propDescriptions = {
     disableWeekends: 'Set to **true** to disables dates that match a weekend.',
     focusOnInit: 'Set to **true** to focus the calendar grid upon being mounted',
     localizedTextShape: {
+        calendarInstructions: 'Localized string informing screen reader users the calendar can be navigated by arrow keys.',
         nextMonth: 'aria-label for next button',
         previousMonth: 'aria-label for previous button',
         show12NextYears: 'aria-label for next button when years are displayed',
         show12PreviousYears: 'aria-label for previous button when years are displayed'
     },
     monthListProps: 'Additional props to be spread to the month\'s `<table>` element.',
+    specialDays: 'Object with special dates and special date types in shape of `{moment(date).format(\'YYYYMMDD\'): type}`. Type must be a number between 1-20.',
```

Comment:
Just glancing at this, it's a little hard to tell how I should be passing a special day in as a prop. I think an example would help (with different types to see some of the different colors from 1-20).

Reply:
I updated the description here and added a few more examples to the component page. Does that seem better or should there be more examples?

Reply:
Yep, the description change and examples make it a lot more clear 👍